ic-agent/src/agent/mod.rs

```diff
@@ -56,7 +56,7 @@ const IC_ROOT_KEY: &[u8; 133] = b"\x30\x81\x82\x30\x1d\x06\x0d\x2b\x06\x01\x04\x
 #[cfg(not(target_family = "wasm"))]
 type AgentFuture<'a, V> = Pin<Box<dyn Future<Output = Result<V, AgentError>> + Send + 'a>>;
 
-#[cfg(target_family = "wasm")]
+#[cfg(all(target_family = "wasm", feature = "wasm-bindgen"))]
```

Comment:
This one should be reverted - it deals with interface rather than implementation. The effect of applying the wasm-bindgen feature should be invisible in the docs.
```suggestion
#[cfg(target_family = "wasm")]
```